test/integration-test.js

```diff
@@ -75,15 +75,6 @@ test('Contributors before a fixed date remain static', t => {
   })
 })
```

Comment:
No new tests?

Reply:
The bulk of the changes are a refactoring, where I used the existing tests to make sure nothing broke. They weren't tied to the old implementation since I always expected to change that.

The only new functionality of any substance is the `filterRepo` function. That can be tested on examples. I'll try to think up some more structural tests as well.

Reply:
Kk.